model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -228,7 +190,7 @@ class PackageCurationTest : WordSpec({
             )
 
             val curation = PackageCuration(
-                id = Identifier.EMPTY,
+                id = pkg.id.copy(type = "${pkg.id.type}Suffix}"),
```

Comment:
Why does this need to be non-empty now, which is a bit of the opposite of the other changes?

Reply:
The test name should explain this: `fail if identifiers do not match`.

Reply:
Ok, but what confused me is that the curation id is derived from the package id by adding a suffix, which seemed like some similarity is needed. As the package id is anyway empty now, can we instead set the curation id to something like `Identifier.EMPTY.copy(type = "Unmatched")` to make the intention explicit?